samples/angular-todo/src/styles/_custom-variables.scss

```diff
@@ -108,7 +108,7 @@ $font-family-sans-serif: 'Open Sans', 'Segoe UI', 'Helvetica Neue', Arial, 'Noto
   'Liberation Sans', sans-serif;
 
 // Override Headers styles and sizes
-$headings-margin-bottom: ($spacer / 2);
+$headings-margin-bottom: ($spacer * 0.5);
```

Comment:
this is because of the slash deprecation in sass. https://sass-lang.com/documentation/breaking-changes/slash-div

I ran the migrator on the sass files that is availble on `npm`

https://sass-lang.com/documentation/breaking-changes/slash-div#automatic-migration